lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -96,28 +96,26 @@ public function getTrimExpression(string $str, int $mode = TrimMode::UNSPECIFIED
 
     /**
      * {@inheritDoc}
-     *
-     * SQLite only supports the 2 parameter variant of this function
```

Comment:
This is obsolete.

Reply:
Do we know since when? We should probably require SQLite to be at least newer than X